CHANGES.md

```diff
@@ -1,7 +1,8 @@
 # 3.11.0 (2021-10-06) - Lille
 
 ## Features/Changes
-* Compiler: setting tc_depth to 0 remove direct call from the tc optimization.  
+* Compiler: setting tc_depth to 0 remove direct call from the tc optimization.
+* Compiler: add --target-env flag, for JS runtime specific compilation targets.
```

Comment:
Can you rebase your PR and move your entry to the appropriate section (unreleased version)